Update print instructions in README.md

RocketUnicorn opened the pull request:
Added printing instructions paragraph related to the M2 machine.

RocketUnicorn committed:
Update README.md

Added printing instructions paragraph related to the M2 machine.

mdtusz commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
 - [Assembly instructions](docs/assembly.md)
 
+## Printing Instructions
+Theses printing instruction are meant for printing on a AON-M2 or an AON-M2-2020.
```
```suggestion

The following instructions are intended for printing using the AON-M2 or an AON-M2-2020.
```

mdtusz commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+Theses printing instruction are meant for printing on a AON-M2 or an AON-M2-2020.
+
+### Printer Preparation
+1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
```
```suggestion

1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
```

mdtusz commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+### Printer Preparation
+1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
+2. Install the aluminum build plate. Ensure that the Kapton tape is in good condition on all the surfaces. (If not, follow [this procedure](https://docs.aon3d.com/maintenance/kapton_application/)to replace it. 
```
```suggestion
2. Install the aluminum build plate. Ensure that the Kapton tape is in good condition on all the surfaces. (If not, follow [this procedure](https://docs.aon3d.com/maintenance/kapton_application/) to replace it. 
```

mdtusz commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+| Extruder   | 250°C       |  
+| Build platform  | 105°C  | 
+| Chamber    | 60°C        | 
+6. Once the chamber has reached thermal equilibrium, initiate the probing sequence on the entire bed  and set the Z offset. Instructions [here](https://docs.aon3d.com/calibration/z_calibration/).
```
```suggestion

6. Once the chamber has reached thermal equilibrium, initiate the probing sequence on the entire bed  and set the Z offset. Instructions [here](https://docs.aon3d.com/calibration/z_calibration/).
```

mdtusz commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+### Frame Printing
+
+We have included two generated G-codes that are ready to upload in your machine for you to start to print as soon as possible. They are intended to be used with ABS on a 0.6 mm nozzle. One of the G-code is for T0 (left toolhead), the other one is for T1 (right toolhead).
```
```suggestion
We have included two generated G-code files that are ready for use on the AON-M2 and AON-M2-2020.
They are intended for use with ABS and a 0.6 mm nozzle.
One G-code file is specific for printing with T0 (left toolhead) and the other is for use with T1 (right toolhead).
```

mdtusz commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+We have included two generated G-codes that are ready to upload in your machine for you to start to print as soon as possible. They are intended to be used with ABS on a 0.6 mm nozzle. One of the G-code is for T0 (left toolhead), the other one is for T1 (right toolhead).
+
+If you wish to modify the files, we also included the factory files to use with Simplify 3D. However, our settings have been tested and any modification can cause print defects.
```
```suggestion
If you wish to modify the files, the original factory files for use with Simplify 3D have been included.
However, the specified settings have been tested and any modification may cause print defects.
```

mdtusz commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+If you wish to modify the files, we also included the factory files to use with Simplify 3D. However, our settings have been tested and any modification can cause print defects.
+
+Each print will get you 5 face shield frames. 
```
```suggestion
Each print will yield 5 face shield frames. 
```

mdtusz commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+- Lower the bed and move the toolhead so you can access the parts easily. **The parts and machines might be hot!**
+- Remove the skirt (single outline of material).
+- With the scraper, carefully detach the extremity of the face shield frame. This will initiate the rest of the removal. Gently detach the part by placing the scraper under the part.
+- Place the parts in a sanitized bag for storage.
```
This section is a bit of a duplicate of what is written [here](https://github.com/aon3d/covid-face-shield/pull/12/files#diff-04c6e90faac2675aa89e2176d2eec7d8R73). Perhaps an edit or two to make it more specific, or migrate the handling procedures down here.

RocketUnicorn replied:
done in : 0d407b5
https://github.com/aon3d/covid-face-shield/pull/12/commits/0d407b5170eecbc5f6190cfadee4c6f8afad5ad7

mdtusz reviewed (commented):
Good additions - just some minor wording and voice changes. I'll let @nugvivi give the final review before full approval. 

:+1:

engnadeau commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
 - Material Safety Data Sheets ([English](docs/msds-en.md) and [French](docs/msds-fr.md)) are available
 - [Assembly instructions](docs/assembly.md)
 
+## Printing Instructions
```
```suggestion
## Printing Instructions

```

linebreak between heading and paragraph for markdown

engnadeau commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+## Printing Instructions
+Theses printing instruction are meant for printing on a AON-M2 or an AON-M2-2020.
+
+### Printer Preparation
```
```suggestion
### Printer Preparation

```

linebreak between heading and paragraph for markdown

engnadeau commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+### Printer Preparation
+1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
+2. Install the aluminum build plate. Ensure that the Kapton tape is in good condition on all the surfaces. (If not, follow [this procedure](https://docs.aon3d.com/maintenance/kapton_application/)to replace it. 
+3. Clean the surface with IPA (isopropyl alcohol) 70% concentration minimum.
```
```suggestion
3. Clean the surface with IPA (isopropyl alcohol), minimum concentration of 70% for disinfection purposes.
```

engnadeau commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+|:----------:|:-----------:|
+| Extruder   | 250°C       |  
+| Build platform  | 105°C  | 
+| Chamber    | 60°C        | 
```
```suggestion
| Chamber    | 60°C        | 

```

engnadeau commented on README.md:
```diff
@@ -124,6 +125,47 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+Plan 112g of ABS filament and 2 hours of print time per print.
+
+
+### Frame Removal
```
```suggestion
### Frame Removal

```

RocketUnicorn committed:
Update README.md

Co-Authored-By: Miklós Tusz <miklos@aon3d.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Miklós Tusz <miklos@aon3d.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Miklós Tusz <miklos@aon3d.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Miklós Tusz <miklos@aon3d.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Miklós Tusz <miklos@aon3d.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Miklós Tusz <miklos@aon3d.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Nicholas Nadeau, Ph.D., P.Eng. <nnadeau@users.noreply.github.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Nicholas Nadeau, Ph.D., P.Eng. <nnadeau@users.noreply.github.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Nicholas Nadeau, Ph.D., P.Eng. <nnadeau@users.noreply.github.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Nicholas Nadeau, Ph.D., P.Eng. <nnadeau@users.noreply.github.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Nicholas Nadeau, Ph.D., P.Eng. <nnadeau@users.noreply.github.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Miklós Tusz <miklos@aon3d.com>

RocketUnicorn committed:
Update frame removal instruction

Added a link to the proper handling guidelines. Removed what was repetitive in the frame removal section.

RocketUnicorn reviewed (commented):
Thanks for your feedback! 
I integrated all your suggestions :)

nugvivi commented on README.md:
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+## Printing Instructions
+
+
+The following instructions are intended for printing using the AON-M2 or an AON-M2-2020.
```
```suggestion
The following instructions are intended for printing using the AON-M2 or AON-M2 2020.
```

nugvivi commented on README.md:
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+1. Install a 0.6 mm nozzle heater block assembly. Instructions [here](https://docs.aon3d.com/maintenance/replace_hba/). 
+2. Install the aluminum build plate. Ensure that the Kapton tape is in good condition on all the surfaces. (If not, follow [this procedure](https://docs.aon3d.com/maintenance/kapton_application/) to replace it. 
+3. Clean the surface with IPA (isopropyl alcohol), minimum concentration of 70% for disinfection purposes.
```
Apply Dimafix or hairspray as well - ABS doesn't stick well enough to clean Kapton without brims. We don't want to use brims so we don't have to post-process

RocketUnicorn replied:
Noted! It will be in the next version.

nugvivi commented on README.md:
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+| Component  | Temperature |  
+|:----------:|:-----------:|
+| Extruder   | 250°C       |  
```
Don't preheat the hot end yet

RocketUnicorn replied:
Removed the extruder line in the table

nugvivi commented on README.md:
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+| Chamber    | 60°C        | 
+
+
+6. Once the chamber has reached thermal equilibrium, initiate the probing sequence on the entire bed  and set the Z offset. Instructions [here](https://docs.aon3d.com/calibration/z_calibration/).
```
Use these probe bounds:
Left: 15
Right: 435
X Count: 10
Front: 15
Back/Rear: 435
Y Count: 10

RocketUnicorn replied:
Added a table with these values

nugvivi commented on README.md:
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+Each print will yield 5 face shield frames. 
+
+Plan 112g of ABS filament and 2 hours of print time per print.
```
```suggestion
Plan 112g of ABS filament and just over 2 hours of print time per print.
```

nugvivi commented on README.md:
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+### Frame Removal
+
+How to remove the part from the build platform. Proceed following the aformentioned [handling(#handling) guidelines. 
```
```suggestion
How to remove the part from the build platform. Proceed following the aformentioned [handling](#handling) guidelines. 
```
i think this needs the second square bracket?

nugvivi commented on README.md:
```diff
@@ -124,6 +125,53 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+How to remove the part from the build platform. Proceed following the aformentioned [handling(#handling) guidelines. 
+
+- Lower the bed and move the toolhead so you can access the parts easily. **The parts and machines might be hot!**
```
The gcode files and factory file should have a command in the ending script to move the tools to the back of the machine

nugvivi replied:
the bed and chamber temps are also left on at the end for quick restart

RocketUnicorn replied:
Don't you think leave the chamber on, opening the door and removing the part will cause thermal runaway issues?

RocketUnicorn replied:
Anyway, I will raise an issue about this but for now I will include your instructions.

RocketUnicorn committed:
Update README.md

Co-Authored-By: Viv <32908428+nugvivi@users.noreply.github.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Viv <32908428+nugvivi@users.noreply.github.com>

RocketUnicorn committed:
Update README.md

Co-Authored-By: Viv <32908428+nugvivi@users.noreply.github.com>

RocketUnicorn committed:
Update printing instruction

Included @nugvivi feedback for more complete print instructions
- Probe bounds
- adhesive
- part removal with heaters remaining hot
- removed preheat of the extruder.

nnadeau committed:
Merge branch 'master' into patch-3

nugvivi commented on README.md:
```diff
@@ -101,7 +104,6 @@ As it seems a person can carry the virus and be asymptomatic, it is best to act
 - Wash your hands before removing prints
 - Wipe down tools with IPA before using them to remove prints
 - Wear a mask while handling prints
-- Touch points on printers (anywhere you might touch the machine) are wiped down regularly with IPA every 4h (e.g., twice a shift)
```
Why remove this?

nugvivi commented on README.md:
```diff
@@ -101,7 +104,6 @@ As it seems a person can carry the virus and be asymptomatic, it is best to act
 - Wash your hands before removing prints
 - Wipe down tools with IPA before using them to remove prints
 - Wear a mask while handling prints
-- Touch points on printers (anywhere you might touch the machine) are wiped down regularly with IPA every 4h (e.g., twice a shift)
```
Why remove this?

RocketUnicorn replied:
I did not recall removing this nor edited the ''handling'' section... What could have happened?

RocketUnicorn replied:
@nnadeau any idea ?

mdtusz replied:
It was Nic's commit here: https://github.com/aon3d/covid-face-shield/pull/12/commits/7848c8cfe22aca8b0f156060050f3739aff7385d 

You didn't do it!

nugvivi commented on README.md:
```diff
@@ -147,6 +149,66 @@ PDFs of the documentation may be found [here](https://github.com/aon3d/covid-fac
+
+**The parts and machines are hot!**
+- At the end of the print, the toolheads will move to the back of the machine for easy access to the printed frames.
+- The toolheads, the build platform and the chamber will remain heated to allow for a quick restart.
```
```suggestion
- The build platform and the chamber will remain heated to allow for a quick restart.
```

nugvivi reviewed (changes requested):
minor correction

RocketUnicorn committed:
Update README.md

Co-Authored-By: Viv <32908428+nugvivi@users.noreply.github.com>
